README.adoc

```diff
@@ -199,6 +199,46 @@ Here are all of the configurable fields in an IOSVariant:
 kubectl apply -n unifiedpush-apps -f ./deploy/crds/examples/push_v1alpha1_iosvariant_cr.yaml
 ----
 
+=== Creating a WebPushVariant for your App
+
+After creating the PushApplication above, you should be able to get
+the `pushApplicationId` from the status, this will be needed to be
+able to create Variants:
+
+[source,shell]
+----
+kubectl get pushApplication example-pushapplication -n unifiedpush-apps -o jsonpath='{.status.pushApplicationId}'
+----
+
+Here are all of the configurable fields in a WebPushVariant:
+
+|===
+|Field Name |Description
+
+|pushApplicationId
+|ID of the PushApplication that this variant corresponds to
+
+|description
+|Human friendly description for the variant
+
+|alias
+|The alias to be used to identify this server.  Must be a url or mailto link
+
+|privateKey
+|The VAPID private key
```

Comment:
What's a VAPID?

Reply:
VAPID is the key format that webpush uses.  It is self registering which means that we don't need to do complicated setup with webpush servers (FCM, WNS, and autopush are all webpush servers that support vapid).

There are online tools to generate keys : https://tools.reactpwa.com/vapid as well as command line tools.

In fact we COULD make the keys auto generate, but I think that it makes more sense to provide them since they will probably be reused by developers during the development cycle and chosen/tested/trusted by ops during deployment.